google-forwarding.md

```diff
@@ -0,0 +1,20 @@
+# Email Forwarding for Gmail Users
+For the best experience, we reccomend that users login directly to their PTA email boxes using the Gmail app on their phone, or in the web browser.  This allows a user to send and recieve email directly from their PTA account.  
```

Comment:
Perhaps the "Gmail app" and "web browser" parts can be markdown links to the respective app stores/websites.